lib/std/fmt.zig

```diff
@@ -807,11 +807,11 @@ test {
 
 pub const Case = enum { lower, upper };
 
-fn formatSliceHexImpl(comptime case: Case) type {
+fn FormatSliceHexImpl(comptime case: Case) type {
```

Comment:
This name (both old and new) has 2 style violations:
* "format" is redundant with "fmt"
* "impl" should never be used since everything is an implementation of something.

Reply:
True, fixed :)